dev/src/types.ts

```diff
@@ -72,6 +72,13 @@ export interface Settings {
   /** Whether to use SSL when connecting. */
   ssl?: boolean;
 
+  /**
+   * The maximum number of idle GRPC channels to keep. A smaller number of idle
+   * channels reduces memory usage but increases request latency for clients
+   * with fluctuating request rates. Defaults to 1.
```

Comment:
Elsewhere you document that setting it to 0 allows the client pool to shut down completely.  Is that a behavior that should be documented here?

Reply:
That's a good suggestion. Added (here and in the other two places...)